tests/testthat/test-digest.R

```diff
@@ -0,0 +1,63 @@
+test_that("sha1.rxUi", {
+  mod1 <- function() {
+    ini({
+      KA=2.94E-01
+      CL=1.86E+01
+      V2=4.02E+01
+      Q=1.05E+01
+      V3=2.97E+02
+      Kin=1
+      Kout=1
+      EC50=200
+    })
+    model({
+      C2 <- centr/V2
+      C3 <- peri/V3
+      d/dt(depot) <- -KA*depot
+      d/dt(centr) <- KA*depot - CL*C2 - Q*C2 + Q*C3
+      d/dt(peri)  <- Q*C2 - Q*C3
+      eff(0) <- 1
+      d/dt(eff)   <- Kin - Kout*(1-C2/(EC50+C2))*eff
+    })
+  }
+
+  ui1 <- rxode2(mod1)
+  ui1$mv0$timeId
+
+  # Wait a few seconds so that compilation will occur at a different time
```

Comment:
@mattfidler, Is there any way to force an `rxUi` object to recompile, even if it is otherwise identical to an existing model? Recompilation seems like a part of making an `rxUi` object different.

Reply:
You can but not with an exported method, and not very well.